Controller/SettingsController.php

```diff
@@ -24,7 +24,10 @@ public function modifyAction(Request $request) {
 			'settings' => $allStoredSettings,
 		);
 
-		$form = $this->createForm(new ModifySettingsForm(), $formData);
+		$form = Kernel::VERSION_ID < 20800
+			? $this->createForm(new LegacyModifySettingsForm(), $formData)
+			: $this->get('form.factory')->createNamed('craue_config_modifySettings', 'Craue\ConfigBundle\Form\ModifySettingsForm', $formData);
```

Comment:
I would rather check for existence of the `getBlockPrefix()` method in the `AbstractType` class (versions of the HttpKernel and the Form component don't necessarily have to be the same).